context.go

```diff
@@ -749,14 +755,19 @@ loop:
 			if tooManyErrors {
 				continue
 			}
-			if blueprintsSet[blueprint.string] {
+			// Limit concurrent calls to parseBlueprintFiles to 200
+			// Darwin has a default limit of 256 open files
```

Comment:
I wouldn't think so - we want at least a few more goroutines than the number of cpus to allow for blocking on IO, but after that I'd expect adding more goroutines to reduce performance, not increase it.  200 is still probably more than we need.